.github/workflows/automatic-release.yml

```diff
@@ -20,10 +20,23 @@ jobs:
     env:
       HAS_CONFIG: false
     steps:
+      - name: Deprecation warning
+        if: ${{ inputs.NODE_VERSION }}
+        run: echo "::warning::The NODE_VERSION input is deprecated and will be removed in the future."
```

Comment:
FYI, the `NODE_VERSION` input is always set because it has a default value, so the warning is always shown.